api/envoy/config/route/v3/route_components.proto

```diff
@@ -1668,6 +1698,9 @@ message RateLimit {
 
       // Rate limit on dynamic metadata.
       DynamicMetaData dynamic_metadata = 7;
```

Comment:
Please deprecate this field, and add a deprecation notice to the release notes. This where I would document the deprecation and tell people to use the new field. If you use proto_format it should sync all the docs for you.

Reply:
Please see my comment above about how I tried to address the deprecation. So I need to add it on the v3 protos too then not just on the v4alpha ones?
I missed the release note thanks for the notice, will address these.

Reply:
Just add it to the v3 file, then run `./tools/proto_format/proto_format.sh fix` and it should sync across the other files.